server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+[[_webauthn]]
+
+=== W3C Web Authentication (WebAuthn)
+
+{project_name} provides the limited support for https://www.w3.org/TR/webauthn/[W3C Web Authentication (WebAuthn)]. {project_name} works as WebAuthn's https://www.w3.org/TR/webauthn/#rp-operations[Relying Party (RP)].
+
+IMPORTANT: Please note that this WebAuthn support is still in development phase and not yet fully completed so that it is recommended to use this WebAuthn support in experimental use. Moreover, there is a possibility that this support's specification and user interfaces will be changed in the future.
+
+NOTE: Whether WebAuthn's operations succeeds depends on a user's WebAuthn supporting authenticator, browser and platform. If you use this WebAuthn support, please clarify to what extent those entities supports WebAuthn specification.
```

Comment:
suggest:  ... supports THE WebAuthn specification.

Reply:
Thank you. I'll fix this sentence as you suggested.